map/src/main/kotlin/edu/artic/map/MapFragment.kt

```diff
@@ -239,6 +237,33 @@ class MapFragment : BaseViewModelFragment<MapViewModel>() {
                 return@setOnMarkerClickListener true
             }
             groundOverlayGenerated.onNext(true)
+
+            /**
+             * Add Lion icons.
+             * TODO:: fix bounds of map.
+             */
+            map.addMarker(MarkerOptions()
+                    .position(LatLng(41.879678006591391, -87.624091248446064))
+                    .icon(bitmapDescriptorFromVector(requireContext(), R.drawable.map_lion_1)))
+
+            map.addMarker(MarkerOptions()
+                    .position(LatLng(41.879491568164525, -87.624089977901931))
+                    .icon(bitmapDescriptorFromVector(requireContext(), R.drawable.map_lion_2)))
+
+        }
+    }
+
+    /**
+     * Builds BitmapDescriptor from vector drawable.
+     */
+    private fun bitmapDescriptorFromVector(context: Context, vectorResId: Int): BitmapDescriptor? {
+        val vectorDrawable = ContextCompat.getDrawable(context, vectorResId)
+        return vectorDrawable?.let { drawable ->
+            drawable.setBounds(0, 0, drawable.intrinsicWidth, vectorDrawable.intrinsicHeight)
```

Comment:
We should also, not put this in a `MapFragment`. This belongs in an `ImageUtils` file as this could be reused in other places. Possibly a `Drawable.toBitmapDescriptorFactory()` extension?